eiffellib/publishers/rabbitmq_publisher.py

```diff
@@ -16,11 +16,13 @@
 """RabbitMQ Eiffel publisher."""
 import time
 import logging
+import warnings
 import pika
```

Comment:
Nit: warnings should go after pika.

Reply:
warnings is in the standard library and pika is an external library

Reply:
Right, of course. But in that case customs call for a blank line between standard library imports and third-party imports.